runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargesConfig.java

```diff
@@ -0,0 +1,68 @@
+package net.runelite.client.plugins.itemcharges;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+		keyName = "itemCharges",
+		name = "Item Charges",
+		description = "Show charge counts remaining on inventory items"
+)
+public interface ItemChargesConfig extends Config
+{
+	@ConfigItem(
+			keyName = "showTeleportCharges",
```

Comment:
This is probably too much indentation, see https://github.com/runelite/runelite/wiki/Code-Conventions.